drivers/dma/dma_dw_common.c

```diff
@@ -70,7 +70,7 @@ void dw_dma_isr(const struct device *dev)
 			 */
 			chan_data->dma_blkcallback(dev,
 						   chan_data->blkuser_data,
-						   channel, 0);
+						   channel, 1);
```

Comment:
I'd like to see the wording around status 1 updated to include something about "block completion" because for DW DMAC that's what this is.